action.yml

```diff
@@ -4,56 +4,41 @@ description: GitHub Action to trigger ci.yml workflow_dispatch event via GitHub
 runs:
   using: composite
   steps:
-
-    - name: Get type
-      id: gettype
+    - name: Derive types
+      id: types
       shell: bash
       env:
         GITHUB_REPOSITORY: ${{ github.repository }}
       run: |
         # repository that has a twice-weekly schedule run on consecutive days - use day of year
-        date_part=$(date +%-j)
-        type=""
+        DATE_PART=$(date +%-j)
```

Comment:
This gha action was using non-standard lowercase variable names. While they are kind of nicer to use, they're also not consistent with the rest of our gha coding style, or just bash coding style in general.  I've refactored them to be uppercase